core/controllers/payload_validator.py

```diff
@@ -153,6 +153,9 @@ def validate(handler_args, handler_args_schemas, allowed_extra_args):
     'NotificationsHandler',
     'OldLibraryRedirectPage',
     'OldNotificationsDashboardRedirectPage',
+    # Oppia Root page is the unified entry for page routes to the frontend.
+    # So, it should exempted from schema validation.
+    'OppiaRootPage',
```

Comment:
This is the list of handlers where the schema will be added in the future. If the schema is not required for this handler then please add it to the HANDLER_CLASS_NAMES_WHICH_DO_NOT_REQUIRE_SCHEMAS constant. Even though it is a unified entry, why should it be exempted from schema validation?

Reply:
done

Reply:
Because the urls with url fragments are also handled with this unified handler. (eg. /about and /learn/<classroom_url_fragment> both are handled using OppiaRootPage handler)

Reply:
Makes sense. Thanks for explanation.